rpmbuild/copr_rpmbuild/builders/mock.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, task, sourcedir, resultdir, config):
         self.buildroot_pkgs = task.get("buildroot_pkgs")
         self.enable_net = task.get("enable_net")
         self.repos = task.get("repos")
+        if task.get("repos") and task.get("repo_priority") is not None:
+            self.repos["priority"] = task.get("repo_priority")
```

Comment:
Are you sure this works? 

In `mock.cfg.j2` we do

```
{% for repo in repos %}
...
priority={{ repo["priority"] }}
```

so it looks that `self.repos["priority"]` isn't the thing that needs to be set. But rather `"repo"` key for each repo.